vagrant/dev/Vagrantfile

```diff
@@ -19,4 +19,33 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   if Vagrant.has_plugin?("vagrant-cachier")
     config.cache.scope = :box
   end
+
+  # This allows the VM to be restarted during provisioning, necessary after
+  # updating the kernel
+  unless Vagrant.has_plugin?("vagrant-reload")
```

Comment:
If you don't have the plugin installed, will the above provisioning that calls it fail? If so, this should be moved before that. Otherwise, here is fine.